lib/runner/cli/argv-setup.js

```diff
@@ -495,6 +495,10 @@ module.exports = new (function () {
         description: 'Set the udid of real iOS device'
       });
 
+      this.option('rerun-failed', {
+        description: 'Rerun failed tests'
```

Comment:
```suggestion
        description: 'Rerun failed test-modules from previous run.

```

Reply:
`Rerun failed tests from previous run`

Modules is confusing.

Reply:
> `Rerun failed tests from previous run`
> 
> Modules is confusing.

@gravityvi opinion?

Reply:
"test suites" is a better fit here.

Reply:
```suggestion
        description: 'Rerun failed test suites'

```